src/frontend/src/views/Upload/components/Metadata/components/ImportFile/parseFile.ts

```diff
@@ -104,7 +104,8 @@ const METADATA_KEYS_TO_EXTRACT = Object.values(HEADERS_TO_METADATA_KEYS);
  *  https://stackoverflow.com/q/64440400
  * So we're just duplicating that aspect here.
  */
-function warnMissingMetadata(
+
```

Comment:
Since this comment is about the below function, there shouldn't be a newline between them